### ISSUE TITLE
Magnifying glass icon in top search can be browsed with VoiceOver but says nothing

### ISSUE THREAD

**Issue by gtritchie** [microsoft/vscode#197576]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.84.1
- OS Version: macOS (I'm using 14.1 but not specific to this version)

Note that this issue is specific to Mac VoiceOver. Not a problem on Windows using NVDA or JAWS .

Steps to Reproduce:

1. Start VoiceOver on macOS via <kbd>Cmd</kbd>+<kbd>F5</kbd>
2. Start Visual Studio code
3. <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>Shift</kbd>+<kbd>Up Arrow</kbd> to move up until "Application" is announced (might take several times depending on starting location)
4. <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>Shift</kbd>+<kbd>Down Arrow</kbd> to enter the "Application", should announce you are on a toolbar (the one with the left/right arrows and search)
5. <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>Shift</kbd>+<kbd>Down Arrow</kbd> to enter the toolbar
6. Navigate forward using VoiceOver cursor until the magnifying glass icon is selected (using <kbd>Ctrl</kbd>+<kbd>Option</kbd>+<kbd>Right Arrow</kbd>)
7. The element is selectable via VoiceOver, but says nothing (no accessible text)

<img width="1032" alt="Screenshot of top toolbar with arrow pointing at the magnifying glass icon" src="https://github.com/microsoft/vscode/assets/10569626/50e98e1a-ec09-414e-a4c8-382b78dda659">

Expected:

I'd lean towards making this icon aria-hidden. It's already that way with other screen readers and you can't just add an aria-label to it (it would need a `role` for that to be fully supported).

**Comment by gtritchie** [comment]
FYI @meganrogge (looks like you do a lot of a11y work).
